rotkehlchen/data_migrations/migrations/migration_10.py

```diff
@@ -24,14 +25,27 @@ def data_migration_10(rotki: 'Rotkehlchen', progress_handler: 'MigrationProgress
     log.debug('Enter data_migration_10')
     with rotki.data.db.conn.read_ctx() as cursor:
         accounts = rotki.data.db.get_blockchain_accounts(cursor)
-    # steps are: ethereum accounts + potentially write to db + updating spam assets
-    progress_handler.set_total_steps(len(accounts.eth) + 3)
+    # steps are: ethereum accounts + potentially write to db + updating spam assets + polygon rpc
+    progress_handler.set_total_steps(len(accounts.eth) + 4)
 
     # Check updates for spam assets. This happens before accounts detection to avoid
     # detecting accounts that only have spam assets.
     progress_handler.new_step('Fetching new spam assets info')
     rotki.data_updater.check_for_updates(updates=[UpdateType.SPAM_ASSETS])
 
+    # we published a data version of the rpc nodes that contained a wrong etherscan name for
+    # polygon. this ensure that the name is correct both in the globaldb and the user db
+    # after the upgrade
+    progress_handler.new_step('Ensuring polygon node consistency')
+    with GlobalDBHandler().conn.write_ctx() as write_cursor:
+        # Update the default nodes, stored in the global DB. It is either correct name
+        # or the bad name since when updating nodes in the globaldb we delete all the nodes
+        write_cursor.execute('UPDATE default_rpc_nodes SET name="polygon pos etherscan" WHERE name="polygon etherscan"')  # noqa: E501
+    with rotki.data.db.conn.write_ctx() as write_cursor:
+        # "polygon pos etherscan" exists because is inserted in the db upgrade. We only need
+        # to delete the bad node if exists
```

Comment:
```suggestion
        # "polygon pos etherscan" exists here because it has been added by the db
        #  upgrade. We only need to delete the bad node if exists
```